Sources/Auth/Services/Common/NetworkingInteractor.swift

```diff
@@ -11,6 +11,7 @@ protocol NetworkInteracting {
     func subscribe(topic: String) async throws
     func unsubscribe(topic: String)
     func request(_ request: RPCRequest, topic: String, tag: Int, envelopeType: Envelope.EnvelopeType) async throws
+    func requestNetworkAck(_ request: RPCRequest, topic: String, tag: Int) async throws 
```

Comment:
We need to get rid of this kind of function that waits for the network acknowledgement (at least if we want to generalize the interactor). We need to find a better way around to wait for the acknowledgement on calls that need it.

Reply:
I am not sure I understand, what you mean by `generalize`? isn't this approach generic and interface could be shared by all SDKs?